src/messaging/Message.ts

```diff
@@ -218,10 +218,34 @@ interface IMessageBodyBase {
   type: MessageBodyType
 }
 
+interface ICostBreakdown {
+  tax: number
+  net: number
+  gross: number
+}
+
+interface IQuote {
```

Comment:
I took the original Quote Timo and I created to include the information Gustav needed